script/setup/critools-version

```diff
@@ -0,0 +1 @@
+0f5f734a7e1da0979915c6e7d5b6641bd9dc2627
```

Comment:
Wondering (not a blocker for me, as this is a test utility); I see the repo does tagged releases; would it be good (and possible) to default to using a tagged version? (gives a bit more context to the reader as to what we're using); https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.20.0

We're currently some commits behind; https://github.com/kubernetes-sigs/cri-tools/compare/0f5f734a7e1da0979915c6e7d5b6641bd9dc2627...v1.20.0 (but perhaps that's intentional)

Reply:
yeah async issue wasn't trying to move up to latest with this PR